website/docusaurus.config.ts

```diff
@@ -0,0 +1,191 @@
+import {themes as prismThemes} from 'prism-react-renderer';
+import type {Config} from '@docusaurus/types';
+import type * as Preset from '@docusaurus/preset-classic';
+import remarkMath from 'remark-math';
+import rehypeKatex from 'rehype-katex';
+
+const GITHUB_URL = 'https://github.com/privacy-scaling-explorations/maci'
+
+const config: Config = {
+  title: 'MACI',
+  tagline: 'Minimal Anti-Collusion Infrastructure',
+  favicon: 'img/maci-logo-2.png',
+
+  // Set the production url of your site here
+  url: 'https://maci.pse.dev',
+  // Set the /<baseUrl>/ pathname under which your site is served
+  // For GitHub pages deployment, it is often '/<projectName>/'
+  baseUrl: '/maci/',
```

Comment:
```suggestion
  baseUrl: '/',
```
I don't _think_ we'll want to specify `/maci/` here, given we'll want to serve this site at `maci.pse.dev` vs. `maci.pse.dev/maci/`

Reply:
removed maci/ as I managed to fix the issue with the custom domain deleting itself